redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/modifiers.kt

```diff
@@ -22,13 +22,13 @@ import app.cash.redwood.schema.LayoutModifier
 /** Grow the item relative to [value] along the main axis. */
 @LayoutModifier(1, scopes = [RowScope::class, ColumnScope::class])
 public data class Grow(
-  val value: Int,
+  val value: Float,
```

Comment:
This an `Shrink` will need docs about the valid range of values. Should it throw outside of [0,1]?

Reply:
Yep, they both need to be positive, `[0, ∞)`. I'll document them for the moment, but eventually we should probably add runtime validation.

Reply:
Oh right. It's like LinearLayout, right? They're portions of a sum.

Reply:
Yep! Exactly